BlueprintUI/Sources/Element/MeasurableStorage.swift

```diff
@@ -17,7 +28,19 @@ extension MeasurableStorage: CaffeinatedContentStorage {
         environment: Environment,
         node: LayoutTreeNode
     ) -> CGSize {
-        measurer(proposal, environment)
+        guard environment.layoutMode.options.measureableStorageCache, let validationKey else {
+            return measurer(proposal, environment)
+        }
+
+        let key = MeasurableSizeKey(path: node.path, max: proposal.maximum)
```

Comment:
Unrelated to your PR, but TIL we store the path as a string, I thought we'd keep the fully qualified `ElementPath` around